src/OrchardCore/OrchardCore.Localization.Abstractions/OrchardCore.Localization.Abstractions.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.ContentManagement.Abstractions\OrchardCore.ContentManagement.Abstractions.csproj" />
```

Comment:
We can't do that or the localization packages would reference content management ones, and right now you can use it in any ASP.NET app.

Reply:
I would suggest `OrchardCore.ContentLocalization.Abstractions`

Reply:
The `ContentItem` is available in `OrchardCore.ContentManagement.Abstractions`, I'm not sure why we should use `OrchardCore.ContentLocalization.Abstractions`?!!

Reply:
@sebastienros waiting for you on this one

Reply:
I just understand why you need to move to `OrchardCore.ContentLocalization.Abstractions`, because `LaocalizationPart` will ensure that Content Localization module is enabled

Reply:
No, it won't.

Reply:
So